### PR TITLE
Added additional guidance to README and console output

### PR DESCRIPTION
**Pull request by nathan-weinberg**
As the tool matures, so too must the README. This is the way.

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -9,11 +9,35 @@ Multiple workloads are run to generate traffic for the the cluster:
...
+```
+
+### Simulating network traffic
+1. Install the [Benchmark Operator](https://github.com/cloud-bulldozer/benchmark-operator) via [Ripsaw CLI](https://github.com/cloud-bulldozer/benchmark-operator/tree/master/cli) by cloning the operator, installing Ripsaw CLI, and running `$ ripsaw operator install`
+2. Once the operator is installed, run the commands below to begin simulating network traffic:
+```bash
+$ source <path/to/uperf_env.sh> # TODO: Determine if we should include this
+$ tmpfile=$(mktemp); envsubst < <path/to/uperf_pod2pod.yaml> > $tmpfile && echo $tmpfile  # TODO: Determine if we should include this
+$ ripsaw benchmark run -f $tmpfile -t 7200
+```
```
Comment (nathan-weinberg) [inline scripts/README.md:36]: @memodi perhaps we can alter these steps to use some of the more generic versions of these files within the `scripts/` directory? any suggestions welcome

Comment (memodi) [reply scripts/README.md:36]: yes, we can keep and example here, for someone who wishes to experiment with NetObserv and create some network traffic. We should highlight that this is an example only though in header of this section.

File: scripts/README.md [hunk 2]
```diff
@@ -9,11 +9,35 @@ Multiple workloads are run to generate traffic for the the cluster:
...
+2. Run the following commands:
+```bash
+oc apply -f scripts/cluster-monitoring-config.yaml
+oc apply -f scripts/service-monitor.yaml
+sleep 10
+oc wait --timeout=120s --for=condition=ready pod -n openshift-user-workload-monitoring -l app.kubernetes.io/component=controller
+oc wait --timeout=120s --for=condition=ready pod -n openshift-user-workload-monitoring -l app.kubernetes.io/managed-by=prometheus-operator
+```
+
+### Simulating network traffic
```
Comment (memodi) [inline scripts/README.md:29]:
```suggestion
### Example simulating pod2pod network traffic
```

File: scripts/README.md [hunk 3]
```diff
@@ -9,11 +9,35 @@ Multiple workloads are run to generate traffic for the the cluster:
...
+1. Install the [Benchmark Operator](https://github.com/cloud-bulldozer/benchmark-operator) via [Ripsaw CLI](https://github.com/cloud-bulldozer/benchmark-operator/tree/master/cli) by cloning the operator, installing Ripsaw CLI, and running `$ ripsaw operator install`
+2. Once the operator is installed, run the commands below to begin simulating network traffic:
+```bash
+$ source <path/to/uperf_env.sh> # TODO: Determine if we should include this
```
Comment (memodi) [inline scripts/README.md:33]:
```suggestion
$ source <path/to/uperf_env.sh> <duration in seconds> # TODO: Determine if we should include this
```

**Review by memodi — CHANGES_REQUESTED** [review]
Overall looks good except for minor changes required.
Added complementing PR: https://github.com/openshift-qe/ocp-qe-perfscale-ci/pull/162 with the example scripts.

**Pull request merged**